test_app/lib/screens/join_screen.dart

```diff
@@ -294,15 +333,22 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
     var params = ConferenceCreateParameters();
     params.dolbyVoice = switchDolbyVoice;
     params.liveRecording = true;
-    var createOptions = ConferenceCreateOption(conferenceName, params, 0);
+    var createOptions = ConferenceCreateOption(
+        conferenceName, params, 0, SpatialAudioStyle.disabled);
+    if (spatialAudioStyleText == spatialAudioWithIndividual) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.individual;
+    } else if (spatialAudioStyleText == spatialAudioWithSharedScene) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.shared;
+    }
```

Comment:
As per my previous comment, would it be possible to keep stuff like `spatialAudioStyleText == spatialAudioWithIndividual` into the `onChanged: (String? value) { }`?

Reply:
I updated the code to remove the check here.